tests/web/test_blocks.py

```diff
@@ -7,9 +7,9 @@
 
 @pytest.mark.bdb
 @pytest.mark.usefixtures('inputs')
-def test_get_block_endpoint(b, client):
```

Comment:
The tests in this file are should be deprecated in favour of `tests/web/test_block_tendermint.py` i.e. no need to update them, just remove them.